### PR TITLE
Changed --delete flag to --delete-changes-without-archiving and modified help messages in archiver command.

### PR DESCRIPTION
**Pull request by ShivanshGahlot**

### CODE REVIEW

File: yb-voyager/cmd/archiveChangesCommand.go [hunk 1]
```diff
@@ -43,9 +43,12 @@ var archiveChangesCmd = &cobra.Command{
 
 func archiveChangesCommandFn(cmd *cobra.Command, args []string) {
 	moveToChanged := cmd.Flags().Changed("move-to")
-	deleteChanged := cmd.Flags().Changed("delete")
+	deleteChanged := cmd.Flags().Changed("delete-changes-without-archiving")
```
Comment (makalaaneesh) [inline yb-voyager/cmd/archiveChangesCommand.go:46]: The `acrhive changes` command already indicates "changes", maybe we keep this flag just `delete-without-archiving` ? @amit-yb

File: yb-voyager/cmd/archiveChangesCommand.go [hunk 2]
```diff
@@ -43,9 +43,12 @@ var archiveChangesCmd = &cobra.Command{
...
 	if moveToChanged == deleteChanged {
-		utils.ErrExit("Either --move-to or --delete flag should be specified")
+		utils.ErrExit("Either --move-to or --delete-changes-without-archiving flag should be set")
+	}
+	if !moveToChanged && deleteSegments.String() == "false" {
+		utils.ErrExit("--delete-changes-without-archiving flag is set to false and --move-to flag is not set. Must set either one of them.")
 	}
```
Comment (amit-yb) [inline yb-voyager/cmd/archiveChangesCommand.go:52]:
```
if moveDestination != "" && deleteSegments {
  utils.ErrExit("only one of the --move-to and --delete-changes-without-archiving should be set")
}
if moveDestination == "" && !deleteSegments {
  utils.ErrExit("one of the --move-to and --delete-changes-without-archiving must be set")
}
```

File: yb-voyager/cmd/archiveCommand.go [hunk 3]
```diff
@@ -42,10 +42,10 @@ func registerCommonArchiveFlags(cmd *cobra.Command) {
 	registerCommonGlobalFlags(cmd)
 
 	cmd.Flags().StringVar(&moveDestination, "move-to", "",
-		"destination to move exported data to")
+		"destination to move exported data to. This also deletes the exported changes from export-dir if the disk utilization threshold is exceeded once the data has been archived.")
```
Comment (amit-yb) [inline yb-voyager/cmd/archiveCommand.go:45]:
```
"Path to the directory where the imported change events are to be moved to. "+
"Note that, the changes are deleted from the export-dir only after the disk utilisation exceeds 70%."
```

File: yb-voyager/cmd/archiveCommand.go [hunk 4]
```diff
@@ -42,10 +42,10 @@ func registerCommonArchiveFlags(cmd *cobra.Command) {
...
 
-	BoolVar(cmd.Flags(), &deleteSegments, "delete", false,
-		"delete exported data after moving it to destination, default is false")
+	BoolVar(cmd.Flags(), &deleteSegments, "delete-changes-without-archiving", false,
+		"delete exported changes without archiving them if the disk utilization threshold is exceeded, default is false")
```
Comment (amit-yb) [inline yb-voyager/cmd/archiveCommand.go:48]:
```
Delete the imported changes without archiving them. Note that: the changes are deleted from the export-dir only after disk utilisation exceeds 70%.
```

File: yb-voyager/cmd/archiveChangesCommand.go [hunk 5]
```diff
@@ -30,9 +30,9 @@ import (
 
 var archiveChangesCmd = &cobra.Command{
 	Use: "changes",
-	Short: "This command will archive the streaming data from the source database.\n" +
+	Short: "This command archives and deletes the streaming data exported from the source database.\n" +
```
Comment (amit-yb) [inline yb-voyager/cmd/archiveChangesCommand.go:33]:
```
Delete the already imported changes and optionally archive them before deleting. 
```

File: yb-voyager/cmd/archiveChangesCommand.go [hunk 6]
```diff
@@ -30,9 +30,9 @@ import (
...
 		"For more details and examples, visit https://docs.yugabyte.com/preview/yugabyte-voyager/reference/cutover-archive/archive-changes/",
-	Long: `This command will archive the streaming data from the source database.`,
+	Long: `This command archives and deletes the streaming data exported from the source database. The data is deleted only if the disk utilization threshold is exceeded. Also, if the --move-to flag is used, only the archived data is deleted.`,
```
Comment (amit-yb) [inline yb-voyager/cmd/archiveChangesCommand.go:35]:
```
This command limits the disk space used by the locally queued CDC events.
Once the changes from the local queue are applied on the target DB (and
source-replica DB), they are eligible for deletion. The command gives an option to
archive the changes before deleting by moving them to some other directory.

Note that: even if some changes are applied to the target databases, they
are deleted only after the disk space utilisation exceeds 70%.
```

**Review by amit-yb — APPROVED** [review]
LGTM. Just make the suggested docs changes and merge the PR.

File: yb-voyager/cmd/archiveChangesCommand.go (hunk 5 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/archiveChangesCommand.go:33]: Done

File: yb-voyager/cmd/archiveChangesCommand.go (hunk 6 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/archiveChangesCommand.go:35]: Done

File: yb-voyager/cmd/archiveCommand.go (hunk 3 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/archiveCommand.go:45]: Done

File: yb-voyager/cmd/archiveCommand.go (hunk 4 above)
Comment (ShivanshGahlot) [reply yb-voyager/cmd/archiveCommand.go:48]: Done

**Pull request merged**